deploy/esignet/install.sh

```
@@ -61,34 +127,130 @@ function installing_esignet() {
+
+  # ---------------------------------------------------------------------
+  # Plugin selection. The keystore type is now derived from this choice:
+  #   mock    -> PKCS12
+  #   mosip   -> PKCS11
+  #   sunbird -> PKCS11
+  # ---------------------------------------------------------------------
+  echo "Please choose the required plugin to proceed with installation"
+  echo "1. mock"
+  echo "2. mosip"
+  echo "3. sunbird"
+  read -p "Enter the plugin number: " plugin_no
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)